HelpSource/Classes/Dictionary.schelp

```diff
@@ -8,7 +8,7 @@ A Dictionary is an associative collection mapping keys to values.
 Two keys match if they are strong::equal::. (i.e. == returns true.)
 
 The contents of a Dictionary are strong::unordered::. You must not depend on the order of items in a Dictionary.
-You must only rely on equality for the keys (e.g. symbols are ok, strings not). For identity matching see: link::Classes/IdentityDictionary::.
+You must only rely on equality for the keys. For identity matching see: link::Classes/IdentityDictionary:: and link::Classes/Event::.
```

Comment:
Thanks!

I think it would be useful to still (properly) mention Strings for equality and Symbol for identity here. I remember these terms were confusing for me when I was learning SC and I think this is a good place to reinforce that difference.

Reply:
Yeah, it was also confusing to me. I change it to something like:
You must only rely on equality for the keys. E.g. symbols and strings can both be used as key because the matching is done by equality (==) and not by identity (===). For identity matching see: link::Classes/IdentityDictionary:: and link::Classes/Event::.

Reply:
I like that. Maybe it's too much but I'd also consider:

You must only rely on equality for the keys. E.g. symbols and strings can both be used as keys because the matching is done by equality (==) and not by identity (===). For identity matching, where strings can not be used as keys, see: link::Classes/IdentityDictionary:: and link::Classes/Event::.

On one hand it's weird because you'd use IdentityDictionary not because you can't use strings, but because you want to match either symbols or actual objects. OTOH this points out that strings won't work for these which is also helpful. What do you think?

Reply:
Yes, sounds good to me. The information about equality vs identity is not so easy to find, so it doesn't hurt to explain it.